package.json

```diff
@@ -14,9 +14,9 @@
     "asar": "^0.13.0",
     "bower": "^1.3.12",
     "electron": "^1.6.1",
-    "electron-builder": "^17.0.1",
+    "electron-builder": "^18.8.1",
     "electron-icon-maker": "^0.0.3",
-    "electron-publisher-s3": "^17.0.2",
+    "electron-publisher-s3": "^19.0.1",
```

Comment:
Wow, I am not a fan of that integrated changelog. `electron-updater` got its own entry, but `electron-publisher-s3` didn't? Confusing. Thanks for digging in for me.

Also really appreciate the forward justification: the features motivating the upgrade in the first place. I think that's really important to include with any module upgrade.

Interesting that it's recommended that we go with prerelease builds. I wondered why they're considered prereleases given that recommendation of wide usage, and I found this: "Wonder why 19.x releases are still pre-releases? Because yet another breaking change is expected (AppX target) :)". Given that they're using heavy-duty semantic versioning tools, I guess we can expect a v20 soon with those changes. Still a bit strange IMO, to map prerelease concepts onto their strong semver adherence. :0/